src/OSPSuite.Presentation/Presenters/ObservedData/BaseDataRepositoryDataPresenter.cs

```diff
@@ -82,13 +78,6 @@ private float convertCellToFloat(int sourceRow, DataColumn observationColumn)
          return observationColumn.Values[sourceRow];
       }
 
-      public string NumericDisplayTextFor(string displayText)
```

Comment:
This is all not required. The extension method from DataBinding takes care of that